client/scripts/views/pages/view_proposal/index.ts

```diff
@@ -436,6 +443,40 @@ const ProposalHeader: m.Component<
                       }),
                   ]
             ),
+            m(MobileToggle, { behavior: 'show' }, [
```

Comment:
If the code in the brackets here and below are identical, you should just factor them into the MobileToggle component above or before the return statement in ProposalHeader. This is so we don't have to change everything twice in case of update.